src/modules/CurrencyModule.cpp

```diff
@@ -18,28 +18,35 @@
 //! Gamble Timeout in seconds
 #define gambleTimeout 20
 
-CurrencyModule::CurrencyModule(UmikoBot* client)
-	: Module("currency", true), m_client(client)
+CurrencyModule::CurrencyModule(UmikoBot* client) : Module("currency", true), m_client(client)
 {
 
 	m_timer.setInterval(24*60*60*1000); //!24hr timer
 	QObject::connect(&m_timer, &QTimer::timeout, [this, client]() 
 		{
-			for (auto server : guildList.keys()) 
+		for (auto server : guildList.keys()) 
+		{
+			//!Clear the daily bonus for everyone
+			for (int i = 0; i < guildList[server].size(); i++) 
 			{
-				//!Clear the daily bonus for everyone
-				for (auto user = guildList[server].begin(); user != guildList[server].end(); ++user
-					) 
+
+				auto user = guildList[server][i];
+				if (UmikoBot::Instance().GetName(server, user.userId) != "") 
 				{
-					user->isDailyClaimed = false;
+					user.isDailyClaimed = false;
 				}
-
+				else 
+				{
+					//remove if the user is no longer in the server
+					guildList[server].removeAt(i);
+				}
+			}
```

Comment:
What happened to the indentation here?

Reply:
The indentation is all fine?

Reply:
Ok it's fine in the code highlighted here, but it's not in the section that follows... On it.